packages/router/src/directives/router_link.ts

```diff
@@ -190,8 +190,8 @@ export class RouterLink implements OnChanges, OnDestroy {
       @Attribute('tabindex') private readonly tabIndexAttribute: string|null|undefined,
       private readonly renderer: Renderer2, private readonly el: ElementRef,
       private locationStrategy?: LocationStrategy) {
-    const tagName = el.nativeElement.tagName;
-    this.isAnchorElement = tagName === 'A' || tagName === 'AREA';
+    const tagName: string|undefined = el.nativeElement.tagName?.toLowerCase();
```

Comment:
```suggestion
    const tagName = el.nativeElement.tagName?.toLowerCase();
```
style nit: I think the type is pretty clear already.